Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/AlternativeIdService.java

```diff
@@ -0,0 +1,75 @@
+package edu.asu.conceptpower.app.core.impl;
+
+import java.util.Collection;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.conceptpower.app.core.IAlternativeIdService;
+import edu.asu.conceptpower.app.core.IConceptTypesService;
+import edu.asu.conceptpower.app.core.impl.ConceptTypesService.ConceptTypes;
+import edu.asu.conceptpower.core.ConceptEntry;
+
+/**
+ * This helper class is used for adding alternative ids to the concept entry.
+ * 
+ * @author karthikeyanmohan
+ *
+ */
+@Service
+public class AlternativeIdService implements IAlternativeIdService {
+
+    @Autowired
+    private IConceptTypesService conceptTypesService;
+
+    /**
+     * This method adds the alternative ids provided concept entry and the id
+     * with which the entry is queried is passed as parameter.
```

Comment:
This sentence isn't really understandable. Do you mean: "This method adds all alternative ids to the given concept entry including the id that was used to search for a concept (passed as method parameter)." ?